test/e2e/specs/test.js

```diff
@@ -0,0 +1,5 @@
+describe('TEST DESCRIPTION', function() {
+  it('SHOULD WORK', function() {
```

Comment:
Can you add a an example using the default frontend bootstrap states? For example, test that when you are on state1, and click on the state2 link, the user is redirected to state2.